modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,95 @@
+Users can now request and receive an export of their personal data within a JSON file, saved to the users personal files.
+The export can be requested any time in selfservice by the user.
+
+=== Secret File Drop
+The feature formerly known as "Uploader" has been renamed to "Secret File Drop", but the functionality stays the same: The Secret File Drop feature allows users to generate unique links that can be shared with external parties. Recipients can anonymously drop files through these links without the need for a registered account or visibility into other submissions. This ensures data privacy and eliminates the risk of unauthorized access.
+Effortless File Collection: With Secret File Drop, collecting files becomes a seamless process. Simply create a file drop link, share it with recipients, and let them submit files directly without any additional steps or complications. This feature saves time and eliminates the need for manual file collection methods, such as email attachments or physical hand-ins.
+
+=== 3 View modes
+Users can now choose three distinct view modes, enhancing file browsing and navigation options:
+Compact List View: The Compact List View provides a condensed and space-efficient representation of your files and folders, designed for users who prefer a streamlined view.
+Regular List View: The Regular List View offers a familiar and comprehensive approach to file organization.
+Tiles View: The Tiles View introduces a visually appealing and intuitive way to browse your files and folders. In this mode, files and folders are displayed as colorful and resizable tiles, providing a visually engaging experience. Users can preview file contents and relevant details directly within the tiles, allowing for quick identification and navigation. Particularly beneficial for users who prioritize visual recognition and prefer a visually rich interface.
+
+=== Trashbins for Spaces
+The Separate Trash Bin for Spaces introduces individual trash bins for each Space within your organization's file system. A space represents a dedicated area where teams collaborate on specific projects, departments, or initiatives. With this feature, accidental file deletions are a thing of the past, as files deleted within a space are now moved to a separate trash bin specific to that space.
+
+=== Deny Access (Experimental - not for production environement)
+The Deny Access feature allows users to share folders with groups, but deny access to a single person in that group. Example: If you share the "Birthday Present" folder with all users in the organization, you want to exclude the one person who has a birthday. This use case is now possible, but not yet production ready and disabled by default. Please contact us and give us your feedback so that we can finalize this long-awaited feature with the help of your testing. 
```

Comment:
```suggestion
The Deny Access feature allows users to share folders with groups, but deny access to a single person in that group. Example: If you share the "Birthday Present" folder with all users in the organization, you want to exclude the one person whose birthday is coming up. This use case is now possible, but not yet production ready and disabled by default. Please contact us and give us your feedback so that we can finalize this long-awaited feature with the help of your testing. 
```